examples/howto/server_embed/django_embed/django_embed/views.py

```diff
@@ -0,0 +1,69 @@
+from os.path import join
+from typing import Any
+
+from django.http import HttpRequest, HttpResponse
+from django.conf import settings
+from django.shortcuts import render
+
+from bokeh.plotting import figure
+from bokeh.layouts import column
+from bokeh.models import ColumnDataSource, Slider
+from bokeh.document import Document
+from bokeh.themes import Theme
+from bokeh.embed import server_document
+
+from bokeh.sampledata.sea_surface_temperature import sea_surface_temperature
+
+from .shape_viewer import shape_viewer
+
+theme = Theme(filename=join(settings.THEMES_DIR, "theme.yaml"))
+
+def shape_viewer_handler(doc: Document) -> None:
```

Comment:
Is the Channels scope and/or Django request object available through this `doc` object? One of the big motivations for integrating Bokeh with Django would be to leverage users and permissions. Access to the currently logged-in user is provided in the Channels scope and/or Django request object. Even if it is available through the `doc` object, it would be better to pass it in as a separate argument.

Reply:
The `doc` is just bokeh's document, but I will add additional arguments to allow accessing the `request` and URL arguments (if provided).